client/src/app/shared/services/login.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from '../models/User';
+
+@Injectable()
+export class LoginService {
+
+  constructor(private http: HttpClient) { }
+
+  login(email: string, password: string ) {
+    return this.http.post<User>('/api/login', {email, password});
```

Comment:
Here, we should save the token for next use ? For now I don't return a user from this request but just a JWT